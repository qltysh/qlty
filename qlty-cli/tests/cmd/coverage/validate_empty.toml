args = [
  "coverage",
  "validate",
  "coverage.zip",
  "--json"
]
bin.name = "qlty"
status.code = 4

[env]
remove = ["GITHUB_ACTIONS"]
