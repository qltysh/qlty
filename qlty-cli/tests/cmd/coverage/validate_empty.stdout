{
  "files_present": 0,
  "files_missing": 0,
  "total_files": 0,
  "coverage_percentage": 0.0,
  "threshold": 90.0,
  "status": "no_coverage_data"
}
